### PR TITLE
Add delete command to got

### PR DESCRIPTION
**Pull request by miguelbernadi**

### CODE REVIEW

**Comment by ifosch** [comment]
+1

**Comment by miguelbernadi** [comment]
~[B]~ There's not a bug

File: mcc/got/got/got.go [hunk 1]
```diff
@@ -109,11 +109,37 @@ func UpsertChangeList(
...
+	list []*route53.ResourceRecordSet,
+) (res []*route53.Change) {
+	var record *route53.ResourceRecordSet
+	for _, name := range names {
+		for _, i := range list {
+			if *i.Name == name &&
+				*i.Type == typ {
+				record = i
+			}
+		}
```
Comment (miguelbernadi) [inline mcc/got/got/got.go:132]: It's not a problem really

**Comment by christofdamian** [comment]
+1

**Pull request merged**